src/inc/quic_platform_winkernel.h

```diff
@@ -162,78 +180,70 @@ CxPlatLogAssert(
     _In_z_ const char* Expr
     );
 
-#define QUIC_WIDE_STRING(_str) L##_str
+#define CXPLAT_WIDE_STRING(_str) L##_str
 
-#define QUIC_ASSERT_ACTION(_exp) \
+#define CXPLAT_ASSERT_NOOP(_exp, _msg) \
+    (CXPLAT_ANALYSIS_ASSUME(_exp), \
     ((!(_exp)) ? \
         (CxPlatLogAssert(__FILE__, __LINE__, #_exp), \
-         __annotation(L"Debug", L"AssertFail", QUIC_WIDE_STRING(#_exp)), \
+         __annotation(L"Debug", L"AssertFail", _msg), \
          DbgRaiseAssertionFailure(), FALSE) : \
```

Comment:
is `DbgRaiseAssertionFailure` platform specific? Maybe it should be moved into the cxplat library? (in a later PR)

Reply:
I believe it's platform specific. But I don't think we need to add an abstraction for it at this time.